.github/workflows/build-depends.yml

```
@@ -127,9 +129,14 @@ jobs:
             depends-${{ hashFiles('contrib/containers/ci/ci.Dockerfile', 'contrib/containers/ci/ci-slim.Dockerfile') }}-${{ inputs.build-target }}-
 
       - name: Build depends
+        env:
+          HOST: ${{ steps.setup.outputs.HOST }}
```

actionlint (1.7.12)

[error] property "setup" is not defined in object type {}

(expression)